bigbluebutton-html5/imports/ui/components/nav-bar/settings-dropdown/component.jsx

```diff
@@ -123,7 +125,8 @@ class SettingsDropdown extends Component {
       />),
     ];
     // Adds Fullscreen button if user is on Android.
-    if (/android/i.test(navigator.userAgent)) {
+
+    if (browser.os === 'Android OS') {
```

Comment:
Why you doesn't pass this to a `isAndroid` variable in the container and uses with props in those if's
 [PR](https://github.com/bigbluebutton/bigbluebutton/pull/5528/files#diff-db9c02403b2d5b44ff3df95904ef349eR59)